### PR TITLE
Qt openGL interaction rewriting

### PR DESCRIPTION
**Pull request by MathiasPaulin**
Solve the "retina" problem but also some performances issues.

### CODE REVIEW

File: src/Engine/Renderer/Renderer.cpp [hunk 1]
```diff
@@ -431,6 +429,7 @@ namespace Ra
             m_pickingFbo->attachTexture( GL_COLOR_ATTACHMENT0, m_pickingTexture.get()->texture() );
             if ( m_pickingFbo->checkStatus() != GL_FRAMEBUFFER_COMPLETE )
             {
+                LOG( logERROR ) << "FBO info : " << m_pickingFbo->checkStatus();
                 LOG( logERROR ) << "FBO Error : " << m_pickingFbo->checkStatus();
             }
```
Comment (hoshiryu) [inline src/Engine/Renderer/Renderer.cpp]: doublon?

Comment (MathiasPaulin) [reply src/Engine/Renderer/Renderer.cpp]: Well, I don't remember this modification ... Nevertheless I'll remove this copy copy/paste side effect on the next commit ... For the moment, this PR is quite perfectible on Linux ... I'm working on that ...

**Comment by nmellado** [comment]
I still have issues on Linux.

When loading a mesh, the camera matrix seems wrong: the screen is all dark gray (looks like there are faces in front of the camera), and when try to rotate the trackball, I can see something moving, like if I was inside the object with a very very wide lens.

Also, 'test0' still generates FBO errors.

**Comment by hoshiryu** [comment]
There is a strange bug when activating x-ray display for bones: wireframe cannot be enabled/disabled unless one resizes the window.

**Comment by nmellado** [comment]
Now properly working on windows with Visual Studio and QtCreator + MinGW

**Comment by nmellado** [comment]
The branch cannot be rebased automatically, I will merge the PR tomorrow.

**Pull request closed without merging**